monorepo/monorepo-ws/apps/update-supergraph-app/src/services/file-services.ts

```diff
@@ -2,19 +2,19 @@ import * as fs from 'fs'
 import {promises as fsPromises} from 'fs';
 import { supergraphFile } from '../configuration/config';
 
-export const checkAndRemoveCreatedFile = (path: string) => {
+export const checkAndRemoveCreatedFile = async (path: string) => {
     try {
         if(!fs.existsSync(path)){
             return;            
         }
-        removeFile(path)
+        await removeFile(path)
     } catch(err) {
         throw new Error(err.message)
```

Comment:
💀 tak za tohle umrelo jedno male roztomile kotatko

Reply:
to ano :(